benchmarks/js/JsBenchmark.js

```diff
@@ -0,0 +1,69 @@
+require('./datasets/google_message1/proto2/benchmark_message1_proto2_pb.js');
```

Comment:
Name this file "js_benchmark.js"?

Reply:
I think we use snake_case names for JS files. Please update this file and other files in this PR as well.